.github/workflows/test-workflow.yml

```diff
@@ -18,6 +18,8 @@ jobs:
           npm ci
         env:
           NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+      - name: Build react icons
```

Comment:
in the same context as the other comment - then it sounds to me like it should be added to each of the commands and not only for the CI.

As a best practice - the CI steps should only be familiar with only one command from the package.json and not try to have some logic composition multiple commands

Reply:
Combined it via `ci` script in `package.json`